src/number_theory/arithmetic_function.lean

```diff
@@ -530,6 +534,45 @@ begin
   apply ((is_multiplicative_zeta).mul is_multiplicative_pow)
 end
 
+/-- `Ω n` is the number of prime factors of `n`. -/
+def card_factors : arithmetic_function ℕ :=
+⟨λ n, n.factors.length, rfl⟩
+
+localized "notation `Ω` := card_factors" in arithmetic_function
+
+@[simp]
+lemma card_factors_one : Ω 1 = 0 := rfl
+
```

Comment:
Maybe this belongs here? I don't know how to prove it.
```
@[simp]
lemma card_factors_eq_one_of_prime {p : ℕ} (hp : p.prime) : Ω p = 1 := sorry
```

Reply:
I've written a bunch of extra lemmas along these lines.